pom.xml

```diff
@@ -11,6 +11,7 @@
     <properties>
         <maven.compiler.source>15</maven.compiler.source>
         <maven.compiler.target>15</maven.compiler.target>
+        <java.version>11</java.version>
```

Comment:
why do we change java version? if it is needed, should we change also in maven.compiler?